ui-library-filesharing-chat-composite/app/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
I'd recommend also removing the web vitals stuff. Keep distractions to a minimum for the reader.